src/common/erc721/ERC721Parser.ts

```diff
@@ -71,7 +69,65 @@ export class ERC721Parser {
         }
     }
 
-    public getContractInstance = async (contractAddress, ABI, ... args: any[]) => {
+    public getContractName = async (contractAddress: string) => {
+        try {
+            const contractPromises = await this.getContractInstance(contractAddress, nameABI)
+            const nameResults = await BluebirdPromise.all(contractPromises).then((names: any) => {
+                const name =  names.filter((name: any) => typeof name === "string" && name.length > 0)
+                return name
+            })
+            let name = nameResults.length > 0 ? nameResults[0] : "";
+            if (name.startsWith("0x")) {
+                name = this.convertHexToAscii(name)
+            }
+            return name;
+        } catch (error) {
+            winston.error(`Error getting contract ${contractAddress} name`)
+            Promise.resolve()
+        }
+    }
+
+    public getContractOwnerOf = async (contractAddress: string, tokenId: string) => {
```

Comment:
@johnnynanjiang Do we use this function for anything ?

Reply:
Not for now, we can remove it.